docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Staging release notes
 
+## November 2023
+
+**09 November 2023**
+
+Frontend version: 1.0.0-20231102.1605 | Backend version: 1.71.2-20231108.1413
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** |  Automated self termination and termination requests by Public officers | In our [service request form](https://go.gov.sg/seed-techpass-support), users can now automate the account termination process for employees who have left the organization. Previously handled manually by L1 and L2 engineers, this process can now be automated to save time. Automation is achieved by calling our Automation API, where the following steps are executed: <br>- Retrieve user information using the user's UPN.<br>- Utilise the `Update user status` endpoint (exclusive to our automation API) to terminate the user's account. <br><br><b>Note</b>: Automation applies only to cases initiated by a Public officer or when the user terminates their account not offboarded to SEED. |
```

Comment:
1. We don't need to divulge our internal implementation.
2. The message intent to the users should be "what's it to them?" Answer: This would immediately terminate the public officer account in a matter of seconds.
3. The Change column can be "Automated self termination by public officers"

Reply:
- 'users can now automate' sounds abit weird, cuz it's not the users who automate, it's the process that has been automated.
- 'this process can now be automated to save time' -> since automation is already in place, instead of saying 'can', we should say 'has now been'?
- Automation applies only to cases initiated by a Public officer (for another user) or when the user themselves terminates their own account, provided the user is not onboarded to SEED.

Reply:
oki!